pyacq/core/manager.py

```diff
@@ -9,46 +9,102 @@ class Manager(RPCServer):
     """
     def __init__(self, name, addr):
         RPCServer.__init__(self, name, addr)
-        self._hosts = {}
```

Comment:
In this case they are not proxies anymore. I think we will only need the proxies in the context of the client / gui, so they are not so useful here. The classes below are only used as data structures for now (might get rid of them).